server/routes/gcp/auth.py

```
@@ -224,6 +224,228 @@ def force_disconnect_gcp(user_id):
+    except Exception as exc:
+        logging.warning(
+            "GCP SA connect: failed to look up secret_ref for user %s (error_type=%s)",
+            user_id,
+            type(exc).__name__,
+        )
+    finally:
+        if cursor is not None:
+            try:
+                cursor.close()
+            except Exception:
+                pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)